operations/rates.md

```diff
@@ -116,9 +226,11 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `IsActive` | boolean | required | Whether the rate is still active. |
 | `IsEnabled` | boolean | required | Whether the rate is currently available to customers. |
 | `IsPublic` | boolean | required | Whether the rate is publicly available. |
+| `Type` | [Rate type](rates.md#rate-type) | required | Type of the rate |
```

Comment:
I'm not clear on the difference between `IsActive`, `IsEnabled` and `IsPublic`. I understood that 'public' means available to end-customers, e.g. through the booking engine, but that is the same definition given for `IsEnabled`?

Reply:
`IsEnabled` - Whether the rate is enabled. Disabled rates aren't applicable - you can set this via UI as well.
`IsPublic` - This is just pure if the `Type` is `Public`
`IsActivate` - This property is on all entities, you can find it already existing in the documentation and meaning of it is always the same. If it's active or not - in other words deleted or not.